StackExchange.Profiling/MiniProfiler.cs

```diff
@@ -359,7 +372,11 @@ public static MiniProfiler FromJson(string json)
 
         private static JavaScriptSerializer GetJsonSerializer()
         {
-            return new JavaScriptSerializer { MaxJsonLength = Settings.MaxJsonResponseSize };   
+            var res = new JavaScriptSerializer { MaxJsonLength = Settings.MaxJsonResponseSize };
+
+            res.RegisterConverters(new[] { new ReadOnlyDictionaryConverter<IReadOnlyList<CustomTiming>>() });
```

Comment:
So this is a total disaster, it looks like JavaScriptSerializer  doesn't support natively IReadOnlyDictionary, so i had to implement and use a converter for it, luckily i don't think we much depend on JavaScriptSerializer except for unit tests. @eugenep-rev, am I correct that we don't rely on JavaScriptSerializer?